Tools/Scripts/libraries/resultsdbpy/resultsdbpy/bug_trackers/bug_description.py

```diff
@@ -0,0 +1,69 @@
+from functools import reduce
```

Comment:
Please put the license in each of these files, you can copy from one of the other `resutlsdbpy` python files and then just edit the date appropriately,